ros_buildfarm/wrapper/apt.py

```diff
@@ -27,6 +27,7 @@ def main(argv=sys.argv[1:]):
         'Hash Sum mismatch',
         'Unable to locate package',
         'is not what the server reported',
+        'corrupted package archive'
```

Comment:
thats a good point fixed, in https://github.com/ros-infrastructure/ros_buildfarm/pull/468/commits/41edac0f82d07190b10be0744733b6a8704a6f5e